### ISSUE TITLE
Support for Symfony 6 

### ISSUE THREAD

**Issue by tacman** [kevinpapst/TablerBundle-Demo#4]
I think the bundle should use Symfony 6 (or Symfony 5.4 with no deprecations).

Once TablerBundle is updated to allow Symfony 6, it's mostly a matter of using using the new security system.

Also, Kernel.php has changed slightly (deprecations).  Using SessionInterface is deprecated, now inject the requestStack and get the Session from there.

**Comment by kevinpapst** [comment]
Yeah, I am recreating the app from scratch. Should be faster than updating all the code and configs manually.

**Comment by tacman** [comment]
Great!  I would love to see the following in the instructions on how to get started with the Bundle, that is, getting started with recreating the demo app:

```
REPO=tabler-demo 
symfony new --full --webapp $REPO && cd $REPO
yarn install && yarn encore dev 
composer req kevinpapst/tabler-bundle
bin/console make:controller AppController

# tweak base.html.twig to use the Tabler layout

# Add some routes to AppController, then add those routes to the menu
```

Ideally, you'd be able to document how to add the js and css to webpack.config.js.

**Comment by tacman** [comment]
Congrats on the release of TablerBundle 1.0!

I've updated my branch of this repo to work with it and Symfony 6.

```bash
git clone git@github.com:tacman/TablerBundle-Demo.git && cd TablerBundle-Demo 
composer install
symfony serve
```

If you can help me squash my (many) commits, I'm happy to make a PR.  Or you mentioned you were doing a new demo app.

**Comment by tacman** [comment]
I tested the demo, works great, thanks.

The only thing that doesn't work is the create-project instruction, since that still pulls the latest release of the demo, I think that will automatically work when a new release is tagged.

**Comment by kevinpapst** [comment]
Ah, never actually tried that command.
Thanks for testing 👍 

I created a new tag and hope it works now.
